core/src/main/scala/org/apache/spark/storage/BlockManagerMasterEndpoint.scala

```diff
@@ -157,7 +158,8 @@ class BlockManagerMasterEndpoint(
       context.reply(true)
 
     case DecommissionBlockManagers(executorIds) =>
```

Comment:
I wanted to confirm my understanding that this PR does not do anything about shuffle files written by executors that have since finished ? That would be the External Shuffle Service's responsibility (if enabled), Right ? And that piece of the puzzle isn't implemented yet ?

My question stems from how we handle this scenario: (Consider the standalone scheduler mode here)
- Executor wrote some shuffle data and is enabled with external shuffle service 
- Executor finished normally and nothing is running on the worker. 
- DecommissionExecutor is not even called because the executor does not exist
- Thus the DecommissionBlockManager isn't called and thus no migrations take place.

Therefore fetch failures would happen for other tasks that try to read this executor's shuffle data (via the External Shuffle Service). I believe fixing this is not in the scope of this PR.

Can you please check this understanding. Thanks.

Reply:
Right we don't do anything in that situation.